ckanext/ontario_theme/templates/internal/snippets/search_form.html

```diff
@@ -9,6 +9,7 @@
     - sr_submit_text
   #}
   <div class="module search">
+    <h2 class="ontario-h2">{{ searchbar_title }}</h2>
```

Comment:
DS H2 is 

```Font: Raleway modified
Size: 27px
Line Height: 1.37
Weight: 700
Spacing: 0.03rem
```
The Figma has the hidden h2 and the search bar title has the properties of 

```font-family: Open Sans;
font-size: 20px;
font-weight: 600;
line-height: 26px;
letter-spacing: 0px;
text-align: left;
```

Reply:
So which should be used?

Reply:
Done. I made a `hidden_searchbar_title` variable that does not have the `:` and I pass this in to the search form template:

```
<h2 aria-label='{{ hidden_searchbar_title }}'><p class="theme_p" aria-hidden="true">{{ searchbar_title }}</p></h2>
```